db/go/20200727144157_add_parameters_to_workflow_template_version.go

```diff
@@ -0,0 +1,82 @@
+package migration
+
+import (
+	"database/sql"
+	v1 "github.com/onepanelio/core/pkg"
+	"github.com/onepanelio/core/pkg/util/pagination"
+	"github.com/pressly/goose"
+)
+
+func initialize20200727144157() {
+	if _, ok := initializedMigrations[20200727144157]; !ok {
+		goose.AddMigration(Up20200727144157, Down20200727144157)
+		initializedMigrations[20200727144157] = true
+	}
+}
+
+func Up20200727144157(tx *sql.Tx) error {
+	// This code is executed when the migration is applied.
+
+	client, err := getClient()
+	if err != nil {
+		return err
+	}
+
+	migrationsRan, err := getRanSQLMigrations(client)
+	if err != nil {
+		return err
+	}
+
+	if _, ok := migrationsRan[20200727144157]; ok {
+		return nil
+	}
+
+	namespaces, err := client.ListOnepanelEnabledNamespaces()
+	if err != nil {
+		return err
+	}
+
+	var pageSize int32
+	var page int32
+	pageSize = 100
+	page = 0
+	paginator := pagination.NewRequest(page, pageSize)
+	wtsResults := -1
+	for _, namespace := range namespaces {
+		for wtsResults != 0 {
+			wts, err := client.ListWorkflowTemplates(namespace.Name, &paginator)
+			if err != nil {
+				return err
+			}
+			//Exit condition; Check for more results
+			wtsResults = len(wts)
+			if int32(wtsResults) > 0 {
```

Comment:
@aleksandrmelnikov Why typecast it to an int32? 

```go
if wtsResults > 0 {
```

Ought to work?

Reply:
```
		wtvsResults = len(wtvs)
```
len returns an `int`.
I could do some more typecasting?

Reply:
I misunderstood, adding change